ksa_compliance/output_models/e_invoice_output_model.py

```diff
@@ -104,6 +106,29 @@ def __init__(
             self.sales_invoice_doc.name, sales_invoice_additional_fields_doc.invoice_doctype
         )
 
+        self.branch_doc = None
+        if self.business_settings_doc.enable_branch_configuration == 1:
```

Comment:
Do a truthiness check instead of comparing to 1 (i.e. `if self.business_settings_doc.enable_branch_configuration`)